steps/check-your-appeal/CheckYourAppeal.js

```diff
@@ -47,8 +47,26 @@ class CheckYourAppeal extends CYA {
         ];
     }
 
-    next() {
+    get form() {
+
+        return form(
 
+            textField('signer').joi(
+                this.content.fields.signer.error.required,
+                Joi.string().regex(whitelist))
```

Comment:
Apparently the user is allowed to sign with an `X`. Silly I know but you may want to speak to Steve & Michael about this. The current implementation doesn't allow for this.

Reply:
changed to the LastName regex to allow 'x'.